packages/_infra/src/index.ts

```diff
@@ -44,8 +45,10 @@ async function main(): Promise<void> {
    */
   const edge = new EdgeStack(basemaps, 'Edge', { env: { region: 'us-east-1', account }, cloudfrontCertificateArn });
   const serve = new ServeStack(basemaps, 'Serve', { env: { region: BaseMapsRegion, account }, albCertificateArn });
+  const cog = new CogStack(basemaps, 'LambdaCog', { env: { region: BaseMapsRegion, account } });
 
   edge.addDependency(serve);
+  edge.addDependency(cog);
```

Comment:
edge doesnt need a dependency on this. this is a seperate url?

Reply:
Fixed.